schemas/fb-linux.yml

```diff
@@ -0,0 +1,56 @@
+---
+- src: "{app_name}_{version}_debian-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+    - arm64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - buster
+        - jessie
+        - stretch
+
+- src: "{app_name}_{version}_ubuntu-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - xenial
+
+- src: "{app_name}_{version}_ubuntu-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+    - arm64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - bionic
+        - focal
+
+- src: "{app_name}-{version}-1.centos-{os_version}.{arch}.rpm"
+  arch:
+    - x86_64
+    - aarch64
```

Comment:
publish action converts arm64 into aarch64 for rpm within the publish action, so assets could still be named as arm64 if required